go.mod

```diff
@@ -89,6 +89,8 @@ require (
 	github.com/heketi/tests v0.0.0-20151005000721-f3775cbcefd6 // indirect
 	github.com/jonboulle/clockwork v0.1.0
 	github.com/json-iterator/go v1.1.8
+	github.com/jstemmer/go-junit-report v0.0.0-20190106144839-af01ea7f8024
```

Comment:
From the list of related K8s PRs, Update SMD to V3 was the last one to do. But this commit is the first in this PR. Are we sure this is the right order?

Reply:
There are 5+ PRs in community before upgrade to SMD V3, and also another 5+ PRs for kube-openapi upgrade. some of them were doing upgrade at the beginning,  and some were doing upgrade at the end. I have merged all these vendor upgrade to one commit to reduce the complication of the PR.